src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/Serialization/IDesignerSerializationProvider.cs

```diff
@@ -27,6 +27,6 @@ public interface IDesignerSerializationProvider
         /// either return it or return null to prevent an infinite
         /// loop.
         /// </summary>
-        object? GetSerializer(IDesignerSerializationManager manager, object currentSerializer, Type objectType, Type serializerType);
+        object? GetSerializer(IDesignerSerializationManager manager, object? currentSerializer, Type? objectType, Type serializerType);
```

Comment:
https://github.com/dotnet/runtime/blob/3a0e52e31c21a648375163776f01076fcb6fcb93/src/libraries/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs#L2206 needs to be updated, too.

Reply:
done.